apps/o2-blocks/package.json

```diff
@@ -1,9 +1,9 @@
 {
-	"name": "@automattic/o2-blocks",
+	"name": "@automattic-ignore/o2-blocks",
```

Comment:
i don't think we want to invent a fake scope. if the intent is not publish this package, we can mark it private.